src/main/java/org/example/Extractor.java

```diff
@@ -9,18 +9,57 @@
 import java.util.List;
 
 public class Extractor {
+    
     public List<Chair> getChairs(String url){
-
-        String role, name, email;
         List<Chair> chairs = new ArrayList<>();
-        //todo
+
+        Document document = getDocument(url);
+        if (document == null) return chairs;
+
+        document.select("table.table>tbody>tr").forEach(tr -> {
+            String role, name = "", email = "";
```

Comment:
It's better to initiate your variables outside the loop.

Reply:
I had this implementation due to the Java constraints of requiring effectively final variables in lambda expressions. I have changed the implementation in my latest commit to address your concerns.

Reply:
Awesome!